core/src/main/java/feast/core/service/AccessManagementService.java

```diff
@@ -75,4 +85,48 @@ public void archiveProject(String name) {
   public List<Project> listProjects() {
     return projectRepository.findAllByArchivedIsFalse();
   }
+
+  /**
+   * List all members registered in the project
+   *
+   * @return List of users in the project
+   */
+  @Transactional
+  public Set<User> listMembers(String project_name) {
+    Optional<Project> project = projectRepository.findById(project_name);
+    if (!project.isPresent()) {
+      throw new IllegalArgumentException(String.format("Could not find project: \"%s\"", project_name));
+    }
+    Project p = project.get();
+    Set<User> result = p.getProjectMembers();
+    return result;
+  }
+
+  @Transactional
+  public void addMember(String user_name, String project_name) {
+    Optional<Project> project = projectRepository.findById(project_name);
+    Project p = project.get();
```

Comment:
Shouldn't only project members be able to grant access to projects to other members?

Reply:
Added a check such that only the project member is allowed to grant access to other members